maintenance/static/css/main.css

```diff
@@ -7375,8 +7375,11 @@ a.text-gray-dark:focus, a.text-gray-dark:hover {
   display: inline-block;
   width: 20px;
   height: 20px;
-  margin-right: 10px;
```

Comment:
I replaced code in this file with what's in `https://foundation.mozilla.org/_css/main.compiled.css` & updated a few image urls to make sure these images show up on maintenance page.